### DISCUSSION TITLE
Not getting context_annotations with API v2

### DISCUSSION THREAD

**Discussion by AnkS4** [tweepy/tweepy#1889] [Questions] [answered]
```
tweets = client.get_users_tweets(
    id = id,
    tweet_fields = ['context_annotations','entities','public_metrics'],
)
```

This is returning only 'id' & 'text':
```

$ tweets.data
[<Tweet id=xxx text='...''>,
 <Tweet id=xxx text='...''>,
...]
```

**Comment by Harmon758** [comment, marked as answer]
> If you are simply printing the objects and looking at that output, the string representations of API v2 models/objects only include the default fields that are guaranteed to exist.
> 
> The objects themselves still include the relevant data, which you can access as attributes or by subscription.

https://tweepy.readthedocs.io/en/v4.9.0/faq.html
